compiler/rustc_codegen_gcc/src/builder.rs

```diff
@@ -1253,8 +1253,9 @@ impl<'a, 'gcc, 'tcx> BuilderMethods<'a, 'tcx> for Builder<'a, 'gcc, 'tcx> {
         self.cx
     }
 
-    fn do_not_inline(&mut self, _llret: RValue<'gcc>) {
+    fn apply_attrs_to_cleanup_callsite(&mut self, _llret: RValue<'gcc>) {
         // FIXME(bjorn3): implement
+        unimplemented!();
```

Comment:
This `unimplemented!()` shouldn't be reintroduced; it was removed in https://github.com/rust-lang/rust/commit/3888aafe3a4a030341fc21f9ac731e1a3bb108f0#diff-e4f794ca308aa712cbad31feb714df7cada1bbe453a30e232e45891571430263L1407-R1255, to work towards proper unwinding support in `rustc_codegen_gcc`.

Doing nothing is a correct (if suboptimal) implementation of this function, since it just adds optimization hints, and doesn't change the semantics of the call.

Reply:
Ok, I just pushed a commit that removes the unimplemented call.